src/main/java/nextstep/image/domain/ImageShape.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.image.domain;
+
+public class ImageShape {
+	private ImageWidth imageWidth;
+	private ImageHeight imageHeight;
```

Comment:
```suggestion
	private ImageWidth width;
	private ImageHeight height;
```
image는 생략해도 되지 않을까요?